service/src/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -326,10 +327,12 @@ public TOpenSessionResp OpenSession(TOpenSessionReq req) throws TException {
 
       final int fetchSize = hiveConf.getIntVar(HiveConf.ConfVars.HIVE_SERVER2_THRIFT_RESULTSET_DEFAULT_FETCH_SIZE);
 
+      Map<String, String> map = new HashMap<>();
```

Comment:
Can we add a test to verify we are getting the config value from server ? Either in TestHiveDatabaseMetaData or a different test ?

Reply:
Sure, I will look into this.

Reply:
Updated the existing test case TestThriftCliServiceWithInfoMessage.java for this. Please take a look, when you get time.